src/libs/pluginprocess/pluginprocess.c

```diff
@@ -6,6 +6,41 @@
  * Executes plugins in a separate process via fork and uses a simple
  * communication protocol based on the dump plugin via named pipes.
  *
+ * The communication protocol works as follows, where Child and Parent stand
+ * for the child and the parent process:
+ * 1)  Two named pipes are created, called commandPipe and resultPipe
+       commandPipe is used for a keyset containing metainformation about
+       the communicationprotocol
+       resultPipe is used for transferring the actual payload that gets passed
+       to a plugin
+ * 2)  The parent constructs a keyset called commandKeySet, containing:
+ *     - /pluginprocess/parent
+ *       a copy of the parent key passed to the plugin interface
+ *     - /pluginprocess/parent/name
+ *       the name of the parent key passed to the plugin interface
+ *     - /pluginprocess/command
+ *       the command that should be executed by the child process
+ *     - /pluginprocess/payload/exists
+ *       1 if a keyset will be sent over the resultPipe, 0 otherwise
+ *     - /pluginprocess/version
+ *       a simple string indicating the version of this communication
```

Comment:
Why not simply using a number?

Reply:
it is a number

Reply:
Perfect, then only the docu needs to be udpated :smile: